src/js/PiskelApi.js

```diff
@@ -133,6 +145,50 @@ var PiskelApi = (function (module) {
   };
 
   /**
+   * Tell Piskel to load an spritesheet for editing, merging with whatever document
+   * is currently open in the editor.  Assumes that frames in the spritesheet
+   * will have a uniform size, and be arranged in a grid read left-to-right,
+   * then top-to-bottom. Will update the final size to be the max of the current or new spritesheet.
+   * @param {!string} uri - An image url, or a data URI corresponding to a
+   *        spritesheet image that Piskel can load.
+   * @param {number} frameSizeX - Width of a frame within the spritesheet, in
+   *        pixels.
+   * @param {number} frameSizeY - Height of a frame within the spritesheet, in
+   *        pixels.
+   * @param {number} [frameRate] - Animation rate in frames per second.
+   * @param {function} [onComplete] - Called when the spritesheet is loaded.
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/data_URIs
+   */
+  PiskelApi.prototype.loadAdditionalFrames = function (uri, frameSizeX, frameSizeY, frameRate, onComplete) {
+    onComplete = typeof onComplete === 'function' ? onComplete : Constants.EMPTY_FUNCTION;
+
+    // Hook up the one-time onComplete callback.
+    var callback = function () {
+      this.removeCallback_(PiskelApi.MessageType.FRAMES_LOADED, callback);
+      onComplete();
+    }.bind(this);
+    this.addCallback_(PiskelApi.MessageType.FRAMES_LOADED, callback);
```

Comment:
It looks like this pattern is used in a couple places (add a callback that gets called once, then removed). I wonder if it's worth adding a helper for it, so that you can just do something like `this.callBackOnce(FRAMES_LOADED, callback)`